src/date/__tests__/monthsAfterStr-test.js

```diff
@@ -0,0 +1,10 @@
+import { expect } from 'chai';
+import monthsAfterStr from '../monthsAfterStr';
+
+describe('monthsAfterStr', () => {
+    it('should return a date string with given number of months ahead of the current date', () => {
```

Comment:
I suggest change the util name to `datePlusNMonth`

And return the date instead of string, since we already provide date to string util

The test description could be 
`It should increase given date by n month`
And it's not clear that this would handle big number of month, just simply add a case where n = 20 ?